libpod/adapter/runtime_remote.go

```diff
@@ -80,21 +83,9 @@ type Container struct {
 
 // remoteContainer ....
 type remoteContainer struct {
-	ID         string
-	Image      string
-	ImageID    string
-	Command    []string
-	Created    time.Time
-	RunningFor string
-	Status     string
-	//Ports            []ocicni.PortMapping
-	RootFsSize int64
-	RWSize     int64
-	Names      string
-	Labels     []map[string]string
-	//	Mounts           []string
-	// ContainerRunning bool
-	//Namespaces       []LinuxNameSpace
+	Runtime *LocalRuntime
```

Comment:
I would really like to call this type "GenericRuntime".

Reply:
PRs welcome!